app/helpers/application_helper/button/new_host_aggregate.rb

```diff
@@ -1,5 +1,13 @@
 class ApplicationHelper::Button::NewHostAggregate < ApplicationHelper::Button::ButtonNewDiscover
+  def supports_button_action?
+    @supported_button_action ||= begin
+      filtered_providers = Rbac::Filterer.filtered(ManageIQ::Providers::CloudManager)
+      filtered_providers.empty? ? false : filtered_providers.all? { |ems| ems.supports?(:create_host_aggregate) }
```

Comment:
Shouldn't this here `any?` instead of `all?`, I assume it's enough to have a single provider that supports *the thing*. Also the ternary operator seems unnecessary to me, is it gaining some extra performance?